versions.tf

```diff
@@ -2,7 +2,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = "~> 3.0"
+      version = "~> 4.0"
```

Comment:
if this module is technically compatible with version 3 and 4 of the provider, what about `>= 3, < 5`?